src/main/java/com/indemnity83/irontank/reference/TankChangerType.java

```diff
@@ -59,10 +51,10 @@
 	 * @return true if upgrade is allowed
 	 */
 	public boolean canUpgrade(TankType from) {
```

Comment:
doesnt this allows ALL upgrades to be applied to ANY type?

Reply:
?

Reply:
What change you say?

Reply:
this in an idea inspection that reverted this also
https://github.com/seitenca/irontanks/blob/support/1.7.10/src/main/java/com/indemnity83/irontank/item/ItemTankChanger.java
!type.canUpgrade -> type.canUpgrade

Reply:
`==` into `!=`